### PR TITLE
Added stub definition to dependency configuration

### PR DESCRIPTION
**Pull request by marcingrzejszczak**
Related to https://github.com/spring-cloud/spring-cloud-zookeeper/issues/37

### CODE REVIEW

File: spring-cloud-zookeeper-discovery/src/main/java/org/springframework/cloud/zookeeper/discovery/dependency/StubsConfiguration.java [hunk 1]
```diff
@@ -0,0 +1,71 @@
+package org.springframework.cloud.zookeeper.discovery.dependency;
+
+import com.google.common.base.Joiner;
```
Comment (spencergibb) [inline spring-cloud-zookeeper-discovery/src/main/java/org/springframework/cloud/zookeeper/discovery/dependency/StubsConfiguration.java]: We shouldn't depend on google classes

File: spring-cloud-zookeeper-discovery/src/main/java/org/springframework/cloud/zookeeper/discovery/dependency/StubsConfiguration.java [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+import lombok.Data;
+import org.apache.commons.lang.StringUtils;
```
Comment (spencergibb) [inline spring-cloud-zookeeper-discovery/src/main/java/org/springframework/cloud/zookeeper/discovery/dependency/StubsConfiguration.java]: Not sure about commons.lang either.

**Comment by marcingrzejszczak** [comment]
@dsyer , @spencergibb - removed all dependencies to Guava and Commons.

**Comment by spencergibb** [comment]
Squashed and merged with some polish via 63ef9101032c798d879542b1538d716b957cbe9a

**Comment by dsyer** [comment]
I guess we need to be careful what we refer to as "Polish" now that @marcingrzejszczak is on the team.

**Comment by spencergibb** [comment]
:laughing: It was lowercase.

**Comment by marcingrzejszczak** [comment]
I've always enjoyed looking at that commit message ;)

**Pull request closed without merging**